packages/api-clients/open-api/bffApi.yml

```diff
@@ -19878,6 +19878,21 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  "/api-docs":
+    get:
+      security: []
+      tags:
+        - docs
+      summary: Swagger docs endpoint
+      description: Return ok
+      operationId: getDocs
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
```

Comment:
is this the intended content type? it's supposed to be for error responses

Reply:
The swagger UI is meant to be viewed in a browser, not consumer by clients, right?
Then I don't think it's useful to track its content type (I suppose it's an HTML which contains links to stylesheets, scripts, etc.?), we can either remove it entirely from this spec or leave it without a specified content type (which is probably legal, I didn't check the OpenApI spec but editor.swagger.io doesn't seem to complain)